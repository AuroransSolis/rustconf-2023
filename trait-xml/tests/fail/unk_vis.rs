trait_xml::trait_xml! {
    <trait>
        <name>Foo</name>
        <vis>bar</vis>
    </trait>
}

fn main() {}
