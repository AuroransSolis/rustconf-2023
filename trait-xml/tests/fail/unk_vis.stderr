error: error parsing visibility: start of invalid visibility specifier `bar`. caller: `::trait_xml::trait_xml_inner`
 --> tests/fail/unk_vis.rs:1:1
  |
1 | / trait_xml::trait_xml! {
2 | |     <trait>
3 | |         <name>Foo</name>
4 | |         <vis>bar</vis>
5 | |     </trait>
6 | | }
  | |_^
  |
  = note: this error originates in the macro `$crate::trait_xml_parse_vis` which comes from the expansion of the macro `trait_xml::trait_xml` (in Nightly builds, run with -Z macro-backtrace for more info)
